final_prep/annotations.rb

```diff
@@ -4,40 +4,126 @@
 
 # Build a Bear
 
+<<<<<<< HEAD
+# define method - establish arguments
 def build_a_bear(name, age, fur, clothes, special_power)
+  # Use code
   greeting = "Hey partner! My name is #{name} - will you be my friend?!"
+  # array - includes name and age of bear
   demographics = [name, age]
+  # expression that bear says
   power_saying = "Did you know that I can #{special_power}?"
+  # Hash for bear characteristics - keys include basic info, clothes, exterior, cost, sayings, and whether it is cuddly
   built_bear = {
+    # series of key - value combos - keys are all strings. values outlined below
+    # value is an array
     'basic_info' => demographics,
+    # value clothes is method parameter
     'clothes' => clothes,
+    # value fur is method parameter
     'exterior' => fur,
+    # value is float
     'cost' => 49.99,
+    # value is array - references two strings from above
+=======
+# define method - establish arguments 
+def build_a_bear(name, age, fur, clothes, special_power)
+  # Use code 
+  greeting = "Hey partner! My name is #{name} - will you be my friend?!"
+  # array - includes name and age of bear 
+  demographics = [name, age]
+  # expression that bear says 
```

Comment:
I encourage you to practice more use of technical terminology and concepts in your annotations; it's a great way to gauge your understanding.